### PR TITLE
IA-4747 update chart locations for apps

### PR DESCRIPTION
**Pull request by calypsomatic**
__Jira ticket__: https://broadworkbench.atlassian.net/browse/IA-4747

## Summary of changes
Adds a liquibase migration to change the chart location for existing apps from the local file directory to the remote terra-helm repository.

### What

-

### Why

- https://github.com/DataBiosphere/leonardo/pull/3987 changed app install (and upgrade!) to pull charts from remote instead of from a local cache.  However, we failed to update the chart location stored in the APP table of the database for already existing apps.  Therefore, when any of those apps are upgraded, helm attempts to pull the chart from the local cache which does not exist anymore.

### What to test

- 

### Who tested and where

- [ ] This change is covered by automated tests 
  - _NB: Rerun automation tests on this PR by commenting `jenkins retest` or `jenkins multi-test`._
- [x] I validated this change by deploying it to my bee and upgrading a WDS.
- [ ] Primary reviewer validated this change 
- [ ] I validated this change in the dev environment

### CODE REVIEW

**Comment by rtitle** [comment]
Note: you also need to update [this file](https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changelog.xml) for the changeset to actually run

File: http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20231214_update_chart_location.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="leonardo" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet logicalFilePath="leonardo" author="bmorgan" id="update-chart-location">
+        <sql dbms="mysql">UPDATE APP
```
Comment (rtitle) [inline http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20231214_update_chart_location.xml:4]: This looks right to me, and I think it's the right set of apps. 👍

**Review by rtitle — APPROVED** [review]

**Comment by calypsomatic** [comment]
jenkins retest

**Comment by calypsomatic** [comment]
problem pulling image, hopefully transient?  jenkins retest

**Comment by calypsomatic** [comment]
I *think* the test failure is unrelated to my change, so I'll try again.  jenkins retest

**Review by cindy-broadinstitute — APPROVED** [review]
LGTM

**Review by mlilynolting — APPROVED** [review]

**Review by lucymcnatt — APPROVED** [review]

**Pull request merged**